R/activate.R

```diff
@@ -78,17 +78,15 @@ renv_activate_impl <- function(project,
     renv_imbue_self(project)
 
   # restart session if requested
-  if (restart) {
+  if (restart)
     renv_restart_request(project, reason = "renv activated")
-  } else {
-    if (renv_rstudio_available())
-      renv_rstudio_initialize(project)
 
-    setwd(project)
+  if (renv_rstudio_available())
+    renv_rstudio_initialize(project)
 
-    # try to load the project
-    load(project)
-  }
+  # try to load the project
+  setwd(project)
+  load(project)
```

Comment:
I needed to change this for interactive testing, as otherwise projects weren't being loaded after `activate` was called.

Reply:
Would it be better to do `restart && !renv_tests_running()`?

Reply:
Makes sense; I made that change.